### PR TITLE
fix(): pass timeout to proxy processing

### PR DESCRIPTION
**Pull request by antmat**

### CODE REVIEW

File: cocaine/proxy/proxy.py [hunk 1]
```diff
@@ -622,8 +622,9 @@ def reelect_app(self, request, app):
         raise gen.Return(app)
 
     @gen.coroutine
-    def process(self, request, name, app, event, data, reelect_app_fn):
-        timeout = self.get_timeout(name, event)
+    def process(self, request, name, app, event, data, reelect_app_fn, timeout = None):
```
Comment (noxiouz) [inline cocaine/proxy/proxy.py]: ```timeout=None```

File: cocaine/proxy/proxy.py [hunk 2]
```diff
@@ -622,8 +622,9 @@ def reelect_app(self, request, app):
...
+        if timeout == None:
```
Comment (noxiouz) [inline cocaine/proxy/proxy.py]: ```if timeout is None```

**Comment by antmat** [comment]
@noxiouz fixed

**Comment by noxiouz** [comment]
LGTM

**Pull request merged**